substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -1717,6 +1717,66 @@ def cinterfacetutorial(args):
     native_image_context_run(_cinterfacetutorial, args)
 
 
+@mx.command(suite.name, 'agenttest', 'Runs the ')
```

Comment:
I feel it would be more accurate to call it `javaagenttest`.
Also, we should complete the usage message here. Something like:
```
usage_msg='Runs tests for java agents with native image'
```

Reply:
Fixed